src/deployment/types.ts

```diff
@@ -50,23 +50,32 @@ export interface MinardDeploymentCreator {
   timestamp: string;
 }
 
-export interface MinardDeployment {
+interface PlainDeployment {
   id: number;
-  commit: MinardCommit;
   commitHash: string;
   ref: string;
   buildStatus: MinardDeploymentStatus;
   extractionStatus: MinardDeploymentStatus;
   screenshotStatus: MinardDeploymentStatus;
   status: MinardDeploymentStatus;
+  projectId: number;
+  projectName: string;
+  teamId: number;
+}
+
+export interface DbDeployment extends PlainDeployment {
+  commit: string;
+  finishedAt?: number | string;
```

Comment:
This depends on whether we are using the sqlite or postgres backend. SQLite stores bigint values as strings.